app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -29,6 +29,7 @@ function show_interface_modal(modal_content) {
   modal_window.modal({'show': true});
 
   modal_window.find('a[rel="popover-modal"]').popover({html: true});
+  $('fieldset.compute_attributes select').select2();
```

Comment:
I agree that we should be consistent as possible in the UI. 
We already have plenty of inconsistencies, I say +1 to applying to all selects pending verification it doesn't break UI because of different sizes and doesn't significantly slow down page load (which should be fairly easy to benchmark).